tests/validation/tests/v3_api/resource/activedirectory.json

```diff
@@ -70,5 +70,34 @@
     "testcommagroup",
     "testuser1",
     "test(san)&abc"
-  ]
+  ],
+  "nested_group_for_ggrb": {
```

Comment:
Can this be renamed to "nested_group_info" since it will be shared across test cases that will  have group related RBAC checks?

Reply:
solved